tests/integration/test_tables.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import pandas as pd
+import pytest
+from google.cloud import bigquery as bq
+
+from bquest.tables import BQTableDefinitionBuilder
+
+pytestmark = pytest.mark.integration
+
+GOOGLE_PROJECT_ID = os.environ.get("GOOGLE_PROJECT_ID")
```

Comment:
You could also use the following code here to retrieve the GCP project_id without having to set the environment variable:

import google.auth as auth
_, project_id = auth.default()